public/img/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Tohle asi bude muset být v rootu, aby to prohlížeč našel (pokud neexistuje nějaký link/meta tag).